packages/nimble-components/src/button/tests/button.mdx

```diff
@@ -21,16 +21,80 @@ import * as buttonStories from './button.stories';
 
 <br />
 
+## Styling
+
+<br />
+
+    ### Ghost Button:
+
+       Ghost is the default style and should be the first consideration for use.
+
+        Use as the default and standard option to create a clean airy and open UI feel. Ghost buttons fit comfortably in tight spaces and help control the visual density of the UI.
+        Be careful when using that the surrounding context does not confuse this button for emphasized body text, tabs or a standalone links.
+
+        Use in combination with a primary outline and primary block buttons to create a hierarchy of importance. There is no primary ghost button.
+
+    ### Outline Button:
+
+        Outline is the secondary style and should be considered for use when ghost button is not sufficient.
+
+        Use as an alternative standard button when a ghost button is not suitable. Use similarly to a ghost button to create a clean, light and airy feel.
+        The outline button is more visually direct about the control’s functionality than a ghost button.
+
+        Use in combination with ghost buttons to create hierarchy. Do not use in combination of a block button to create hierarchy.
```

Comment:
There are a few places that say "in combination of", which probably should be "in combination with". But that change results in very repetitive text. Maybe these should be like "Use in combination with ghost buttons (but not block buttons) to create a hierarchy of importance."